pkgs/models/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, recurseIntoAttrs, stdenvNoCC, fetchurl, makeWrapper, sticker }:
+
+let
+  stickerModel = import ./model.nix;
+
+  # Embeddings fetcher, uses the sticker-models repository.
+  fetchEmbeddings = { name, sha256 }: {
+    embeds = fetchurl {
+      inherit sha256;
+      url = "https://github.com/stickeritis/sticker-models/releases/download/${name}/${name}.fifu";
+    };
+
+    filename = "${name}.fifu";
+  };
+
+  deWordEmbeds = fetchEmbeddings {
```

Comment:
This is an invocation of the above `fetchEmbeddings` which defines `deWordEmbeds`, referring to `https://github.com/stickeritis/sticker-models/releases/download/de-structgram-20190426-opq/de-structgram-20190426-opq.fifu`?

Reply:
Indeed.